cli/popper/scm.py

```diff
@@ -183,6 +183,12 @@ def get_default_branch(r):
         else:
             url += '/'
 
+        auth_token = os.getenv('GITHUB_API_TOKEN')
+
+        if(auth_token is not None):
```

Comment:
can we add logic for checking that the URL is pointing to github? If the URL is pointing somewhere else it'll likely fail.

Reply:
Yeah, sure I can add it.